src/modules/Bookmark/controller.ts

```diff
@@ -42,4 +43,37 @@ const createBookmark = async (req: Request, res: Response) => {
   }
 };
 
-export default { createBookmark };
+const getBookmarkedArticle = async (
+  req: Request,
+  res: Response,
+): Promise<void> => {
+  const { bookmarkId } = req.params;
```

Comment:
We will usually get the bookmarks per User, each user should be able to see which News He bookmarked;
So a proper implementation for this can be:

- Get User from Request
- Search by User in the Bookmark Table and Return all Bookmarks

Reply:
I agree, so, if you confirm, then I would remove my changes for getting one bookm arked article by id and create a new issue for returning all bookmarks by user

Reply:
Confirmed
!